nixos/modules/services/monitoring/prometheus/exporters/smartctl.nix

```diff
@@ -4,16 +4,7 @@ with lib;
 
 let
   cfg = config.services.prometheus.exporters.smartctl;
-  format = pkgs.formats.yaml {};
-  configFile = format.generate "smartctl-exporter.yml" {
-    smartctl_exporter = {
-      bind_to = "${cfg.listenAddress}:${toString cfg.port}";
-      url_path = "/metrics";
-      smartctl_location = "${pkgs.smartmontools}/bin/smartctl";
-      collect_not_more_than_period = cfg.maxInterval;
```

Comment:
Looks like you removed this option. Use `mkRemovedOptionModule`.

Reply:
That's not an option, that's the content of the config file which I migrated to cli arguments. There is no config file anymore.

Reply:
I missed that you migrated that option, so I resolved this review.